pageserver/src/walredo.rs

```diff
@@ -170,22 +170,21 @@ impl WalRedoManager for PostgresRedoManager {
 }
 
 fn mx_offset_to_flags_offset(xid: MultiXactId) -> usize {
-    return ((xid / pg_constants::MULTIXACT_MEMBERS_PER_MEMBERGROUP as u32) as u16
+    ((xid / pg_constants::MULTIXACT_MEMBERS_PER_MEMBERGROUP as u32) as u16
         % pg_constants::MULTIXACT_MEMBERGROUPS_PER_PAGE
-        * pg_constants::MULTIXACT_MEMBERGROUP_SIZE) as usize;
+        * pg_constants::MULTIXACT_MEMBERGROUP_SIZE) as usize
 }
```

Comment:
Using `as` to truncate integers is probably over-used in our code. It would be nice to see `try_into()` used more often instead (even if it's just `.try_into().unwrap()`; at least we won't see the bad value propagating further.